### ISSUE TITLE
Improve state conservation status 

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/biocache-store#142]
When indexing a record, the conservation status source list UID should be recorded along with the value so that the user can click through to the conservation list in the specieslist app.

Some records have multiple values for state conservation (processed). Does this make sense or should there be only a single list for each state?

E.g.http://biocache.ala.org.au/ws/occurrences/16c8f940-9b88-4c56-b2fd-a7b256b973fd

which has `stateConservation: "Endangered,Vulnerable"`

**Comment by nickdos** [comment]
Also - check with Michael for list of issues raised in the Collections consultation meeting in Feb 2016, for any related issues to conservation status.

**Comment by djtfmartin** [comment]
Good idea on storing the UID for this species list. We can probably do this in the taxon table in cassandra.

The 2 values presented (badly) are status and sourceStatus - where sourceStatus is the verbatim status supplied in the conservation list.

**Comment by nickdos** [comment]
Niels has added a lot more details (AVH use case) here: https://github.com/AtlasOfLivingAustralia/avh-hub/issues/57

**Comment by nielsklazenga** [comment]
That was actually a lot of waffle before I had worked out what was really going on. The only problem is the concatenation of the 'sourceStatus' and 'status' from the lists (as Dave says). If this could just be split up into separate fields it will work fine. I noticed that there are `country_conservation`, `raw_country_conservation`, `state_conservation` and `raw_state_conservation` in the index, so the intention seems to have been there, but the at least the `raw_country_conservation` field is not populated.

@djtfmartin : Do you have any recollection how the 'status' field was derived (and why)?

**Comment by nickdos** [comment]
I was thinking about this at the end of the HISCOM teleconference and a short term solution might be to simply provide a better label to the user via i18n lookup for each value. The only hard part is finding the full list of possible values - which a facet search should do.
